device/globals/terrain.gd

```diff
@@ -9,6 +9,8 @@ export(int, "None", "Scales", "Lightmap") var debug_mode = 1 setget debug_mode_u
 export var lightmap_modulate = Color(1, 1, 1, 1)
 export var scale_min = 0.3
 export var scale_max = 1.0
+export var player_speed_multiplier = 1.0  # Compensate if you want a small player to move fast
```

Comment:
Could change the comment to something like "override player speed in current scene", since there might be use cases where one would want to increase speed not just because the player is smaller.

Reply:
Done, force-pushed.